packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.8.0
+
+* Adds support for the native `onRenderProcessGone` callback.
```

Comment:
nit: It's preferred to have a more descriptive explanation of the feature and the specific method added. Eg.

```
* Adds support to receive a notification when a WebView's render process has exited. See 
  `AndroidNavigationDelegate.setOnRenderProcessGone`.
```